### PR TITLE
Remove journald path from the fluent-bit and promtail configuration.

### LINKED ISSUE #6965: Gardener Logging stack could not properly detect whether the journald logs in /var/log/journal or /run/log/journal
**How to categorize this issue?**

/area logging
/kind bug

**What happened**:
People often complain that the `systemd` logs from SUSE machines are missing.
The `/var/lib/promtail/scripts/set_active_journal_file.sh` script as part from the `promtail` systemd service decides where on the filesystem - either from `/run/log/journal` or from `/var/log/journal` - promtail should parse the system logs. If the `/var/log/journal` exists it is preferred over  `/run/log/journal`. And, although the `/var/log/journal` dir exists then `systemd` logs are stored under `/run/log/journal`.

According to the @D060239 [investigation](https://github.tools.sap/kubernetes-live/issues-live/issues/2021#issuecomment-1487992)  the `/var/log/journal` at early boot time  existed, but was not mounted r/w, so the directory check on `/var/log/journal` during the boot process made by `journald` failed. Thus the `journald` logs to `/run/log/journal` and `promtail` looks fo those logs in /var/log/journal`.

**What you expected to happen**:
Gardener should correctly identify the logging directory for `systemd-journald` and not rely on a directory existence check.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
As a first step, let's try to duplicate the `promtail's` `journald` configuration in order to try to scrape `systemd` logs both from `/var/log/journal` and `/run/log/journal`.
**Environment**:

- Gardener version:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:

### PR DESCRIPTION
**Pull request by vlvasilev**
**How to categorize this PR?**

/area logging
/kind enhancement

**What this PR does / why we need it**:
It looks like `Fluent-Bit` and `Promtail` have an efficient mechanism to detect in which directory `journald` logs. By removing the explicit path both log processor manage to choose between `/var/log/journal` and `/run/log/journal`.

**Which issue(s) this PR fixes**:
Fixes [#6965](https://github.com/gardener/gardener/issues/6965)

**Special notes for your reviewer**:

**Release note**:

```bugfix operator
Gardener correctly identifies the logging directory for `systemd-journald` and does not rely on a directory existence check.
```

### CODE REVIEW

File: pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/promtail/component_suite_test.go [hunk 1]
```diff
@@ -19,9 +19,11 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
 )
 
 func TestPromtail(t *testing.T) {
+	format.MaxLength = 0
```
Comment (rfranzke) [inline pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/promtail/component_suite_test.go]: Was this just a temporary change that should be reverted?

**Review by rfranzke — COMMENTED** [review]
/cc @nickytd @Kristian-ZH

**Review by rfranzke — COMMENTED** [review]
Can you drop the unrelated commits c313e00e44e7437cc776152b0245225e3ab39b9a and 392a9fed0ca7b126170c40ea32ec6b382f70f251 from this PR?

**Review by rfranzke — COMMENTED** [review]
/approve

**Review by Kristian-ZH — APPROVED** [review]
/lgtm

**Pull request merged**